src/librados/IoCtxImpl.cc

```diff
@@ -1518,7 +1494,7 @@ int librados::IoCtxImpl::stat(const object_t& oid, uint64_t *psize, time_t *pmti
 
   ::ObjectOperation rd;
   prepare_assert_ops(&rd);
-  rd.stat(psize, &mtime, NULL);
+  rd.stat(psize, &mtime, static_cast<int*>(0));
```

Comment:
why not nullptr?

Reply:
To disambiguate.

Since we have one overload that's `(std::uint64_t*, ceph::real_time*, int*)` and one that's (std::uint64_t*, ceph::real_time*, boost::system::error_code*)

Reply:
+1, though I'm wondering if changing the int* signature to nullptr_t would be useful. A comment in the code might be good here, too.

Reply:
or just adding a separate tagged interface (e.g. stat(something, something, ErrorCodeTag, boost::system::error_code*) ?

Reply:
Fixed in next push!